draft-irtf-cfrg-bbs-signatures.md

```diff
@@ -1668,11 +1660,17 @@ Applications using the Interface defined in (#bbs-signatures-interface), MUST ig
 
-An option is for the Issuer to publish this information as part of their public parameters, similar to TBD (U-Prove). Such configuration should detail the type of each message, based on that message's index on the signed messages list (i.e., the first message will be an octet string, the second an integer etc.). A BBS Interface should check the messages they receive against those configurations and map them to scalars accordingly. Another option is to sign such configurations as part of the header parameter of the BBS signature (see (#signature-generation-sign)). In this case, the configuration does not need to be published by the Issuer. The Prover will be responsible to get that information from the issuer and later, to communicate it to the Verifier.
+An option is for the Issuer to publish this information as part of their public parameters, similar to [@UPROVE]. Such configuration should detail the type of each message, based on that message's index on the signed messages list (i.e., the first message will be an octet string, the second an integer etc.). A BBS Interface should check the messages they receive against those configurations and map them to scalars accordingly. Another option is to sign such configurations as part of the `header` parameter of the BBS signature (see (#signature-generation-sign)). In this case, the configuration does not need to be published by the Issuer. The Prover will be responsible to get that information from the issuer and later, to communicate it to the Verifier.
+
+## Post-quantum Security and Everlasting Privacy
+
+The BBS Signatures scheme has two main security properties; unforgeability of BBS signatures and zero-knowledge of BBS proofs. The first property, referring to the inability of anyone other than the Signer to be able to generate BBS signatures, does not hold true against an adversary using a strong enough quantum computer. Specifically, such a computer could be used to reveal the Signer's secret key from their public key, hence giving them the ability to generate BBS signatures on behalf of that Signer for any message they want.
+
+On the other hand, the second security property, i.e., the zero-knowledge of the BBS proof, cannot be broken, even by utilizing a quantum computer. We refer to this property as "everlasting privacy". This guarantees that an adversary in possession of a quantum computer, will not be able to reveal neither the messages undisclosed by a BBS proofs, nor the hidden signature value. As a result, the privacy and hiding properties of the BBS proofs that are currently used, will not be compromised by future quantum-attacks.
```

Comment:
you probably want to use terminology from PQUIP here.

https://datatracker.ietf.org/doc/draft-ar-pquip-pqc-engineers/

CRQCs, etc.

Reply:
Than you for the suggestion! Updated.